packages/dai-plugin-mcd/scripts/prepublish.sh

```diff
@@ -4,6 +4,20 @@ if [ $(basename $(pwd)) != "dai-plugin-mcd" ]; then
   echo "This script must be run from the dai-plugin-mcd directory."
   exit
 fi
+
+yarn config set version-tag-prefix "dai-plugin-mcd-v"
+yarn config set version-git-message "dai-plugin-mcd-v%s"
+yarn version
+
+PACKAGE_VERSION=$(cat package.json \
```

Comment:
i'm ok introducing a dependency on `jq` because it is such a handy CLI tool anyway. this just becomes `jq .version package.json` in that case

Reply:
This actually isn't necessary anymore, since I removed the pushing. Good to know though.